addons/binding/org.openhab.binding.zoneminder/src/main/java/org/openhab/binding/zoneminder/handler/ZoneMinderServerBridgeHandler.java

```diff
@@ -394,111 +365,140 @@ public void handleCommand(ChannelUID channelUID, Command command) {
+                    if (curPriority.isPriorityActive(getBridgeConfig().getDiskUsageRefresh())) {
+                        IZoneMinderDiskUsage diskUsage = null;
+                        try {
+                            diskUsage = zoneMinderServerProxy.getHostDiskUsage();
+                            logger.debug("{}: URL='{}' ResponseCode='{}' ResponseMessage='{}'", getLogIdentifier(),
+                                    diskUsage.getHttpRequestUrl(), diskUsage.getHttpStatus(),
+                                    diskUsage.getHttpResponseMessage());
+                        } catch (Exception ex) {
+                            logger.error(
+                                    "{}: context='refreshThing' Exception thrown in call to ZoneMinderDiskUsage ('{}')",
+                                    getLogIdentifier(), ex.getMessage());
+                        } catch (ZoneMinderException ex) {
+                            logger.error(
+                                    "{}: context='refreshThing' error in call to 'getHostDiskUsage' ExceptionClass='{}' - Message='{}'",
+                                    getLogIdentifier(), ex.getClass().getCanonicalName(), ex.getMessage(),
+                                    ex.getCause());
+                        }
 
-                if (diskUsage == null) {
-                    logger.warn("{}: ZoneMinderDiskUsage dataset could not be obtained (received 'null')",
-                            getLogIdentifier());
-                } else if (hostLoad.getHttpResponseCode() != 200) {
-                    logger.warn(
-                            "{}: ZoneMinderDiskUsage dataset could not be obtained (HTTP Response: Code='{}', Message='{}')",
-                            getLogIdentifier(), hostLoad.getHttpResponseCode(), hostLoad.getHttpResponseMessage());
+                        if (diskUsage == null) {
+                            logger.warn(
+                                    "{}: context='refreshThing' ZoneMinderDiskUsage dataset could not be obtained (received 'null')",
+                                    getLogIdentifier());
+                        } else if (diskUsage.getHttpStatus() != HttpStatus.OK_200) {
+                            logger.warn(
+                                    "{}: context='refreshThing' ZoneMinderDiskUsage dataset could not be obtained (HTTP Response: Code='{}', Message='{}')",
+                                    getLogIdentifier(), diskUsage.getHttpStatus(), diskUsage.getHttpResponseMessage());
 
-                } else {
-                    channelDiskUsage = diskUsage.getDiskUsage();
+                        } else {
+                            channelDiskUsage = diskUsage.getDiskUsage();
+                        }
+                    }
                 }
+            } else {
+                // Make sure old data is cleared
+                channelCpuLoad = "";
+                channelDiskUsage = "";
+            }
+        } catch (Exception ex) {
```

Comment:
Please catch specific exceptions instead of catching all exceptions. That way it is more clear what exceptions need to be handled and how this is done.